IO-pc-local/src/app/lobby/lobby.service.ts

```diff
@@ -1,26 +1,43 @@
 import { Injectable } from '@angular/core';
-import { BehaviorSubject, Observable } from 'rxjs';
+import { Subject, Observable } from 'rxjs';
 import { Student } from './shared/model';
 
 @Injectable({
   providedIn: 'root'
 })
 export class LobbyService {
-  private readonly _students: BehaviorSubject<Student[]> = new BehaviorSubject([] as Student[]);
-  public students$: Observable<Student[]> = this._students.asObservable();
+  public readonly students = new Subject<Student[]>();
   private sse: EventSource | null = null;
```

Comment:
Nie musi być inicjalizowane null'em, jeśli wartość jest ustawiana w konstruktorze, a więc może być:
```
private readonly sse: EventSource;
```

Dodatkowo sama inicjalizacja może być już tutaj, ale to już kwestia gustu. Wtedy byłoby:
```
private readonly sse: EventSource = new EventSource("http://localhost:8000/game/lobby",
    { withCredentials: true, });
```